README.md

```diff
@@ -115,79 +118,57 @@ If you are _not_ using the shaded Bigtable HBase Client artifact, you need to de
 If you _are_ using the shaded Bigtable HBase Client artifact, then the OpenCensus dependencies are embedded in the
 shaded artifact; i.e. nothing additional for you to do.
 
+[//]: # ({x-version-update-start:bigtable-client-parent:released})
 ```xml
 <!-- OpenCensus dependencies -->
 <dependency>
     <groupId>com.google.cloud.bigtable</groupId>
     <artifactId>bigtable-hbase-1.x-shaded</artifactId>
-    <version>2.0.0-alpha1</version>
+    <version>2.0.0</version>
```

Comment:
please replace with latest bigtable version (separately we should look into why this isn't getting updated automatically)